base/acme/src/main/java/org/dogtagpki/acme/server/ACMEEngineConfigFileSource.java

```diff
@@ -28,6 +28,7 @@
     static org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(ACMEEngine.class);
 
     String filename;
+    boolean monitor = true;
```

Comment:
The initialization is not necessary since it will be done again in `init()`, but it's no big deal.